src/styles.css

```diff
@@ -0,0 +1,1209 @@
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 4px 0.5px;
+}
+
+.project-description {
+  margin-left: 5%;
+}
+
+.project-description > h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.project-details {
+  width: 100%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 5%;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.skills {
+  display: flex;
+  float: right;
+  align-items: center;
+  position: static;
+  padding: 0.5px;
+  width: 100%;
+  height: 24px;
+  left: 0.5px;
+  top: 180px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin-top: 4%;
+}
+
+.skill {
+  position: static;
+  width: 29px;
+  height: 16px;
+  left: 12px;
+  top: 4px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  margin: 0.5px 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.button-click {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.button-resume {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 24px 0.5px;
+  padding-left: 20px;
+}
+
+.btn {
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  padding: 7px;
+  top: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-top: 5%;
+}
+
+.btn:hover {
+  background-color: #396df2;
+  color: #fff;
+}
+
+button:active {
+  background-color: #2230d2;
+}
+
+.btn:active {
+  background-color: #2230d2;
+}
+
+.about {
+  background: #fff;
+  width: 70%;
+  padding: 64px;
+  border-top-right-radius: 55px;
+}
+
+.about-me {
+  display: flex;
+  position: static;
+  width: 40%;
+  left: 0.5px;
+  top: 0.5px;
+  right: auto;
+  padding: 15px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40.5px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0.5px;
+}
+
+.lang {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  padding: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.imglang {
+  width: 30px;
+  padding-right: 40px;
+}
```

Comment:
- [ ] Kindly make sure to style about language section to match that of the `Figma` design as it's going to help make your design look professional and create a good user experience. See below for more details:
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="372" alt="Screenshot 2023-07-06 at 16 14 30" src="https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/107192076/20df85d4-6e6a-4cd8-a877-2bee1328cae1">
</td>
<td>
<img width="345" alt="Screenshot 2023-07-06 at 16 12 43" src="https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/107192076/e1bf9a1d-f358-412c-9585-600fcf6a4600">
</td>
</tr>
</table>